src/data/context/apyChartContext.tsx

```diff
@@ -166,9 +162,15 @@ export const ApyChartProvider: FC<{
     apy: true,
   })
   const [timeline, setTimeline] = useState<Timeline>("1W")
-  const launchDate = Object.values(cellarDataMap).find(
+  const cellarData = Object.values(cellarDataMap).find(
     (item) => item.config.cellar.address === address
-  )?.launchDate!
+  )
+  const launchDate = cellarData?.launchDate!
+  const [{ data: sgData }] = useGetStrategyDataQuery({
+    variables: { cellarAddress: address.toLowerCase() },
+  })
+  const getDecimals = sgData?.cellar?.asset.decimals ?? 18
```

Comment:
non blocking the merge but this "get..." name is a variable not a function. Better rename this to `sgDecimals`

Reply:
fallback value better `6` because we have only one strategy that underlying asset is eth